docs/develop/java/spring/redis-om/index-redis-om-spring.mdx

```diff
@@ -0,0 +1,53 @@
+---
+id: redis-om-spring
+title: Redis OM - Spring
+sidebar_label: Introduction
+slug: /develop/java/spring/redis-om/redis-om-spring
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import RedisCard from '@site/src/theme/RedisCard';
+
+
+## Introduction
+
+The Redis OM family of projects aim is to provide high-level abstractions idiomatically implemented for
+ your language/platform of choice. We currently cater to the Node, Python, .Net and Spring communities.
+
+The Spring Framework is the leading full-stack Java/JEE application framework and Redis OM Spring (ROMS)
+goal is to enable developers to easily add the power of Redis to their Spring Boot applications.
+
+Redis OM Spring provides powerful repository and custom object-mapping abstractions built on top of the
+powerful Spring Data Redis (SDR) framework.
```

Comment:
"powerful" is used twice. Maybe use something like "capable" or something for one of them.